### DISCUSSION TITLE
Weekly Governance Updates (13 January, 2023)

### DISCUSSION THREAD

**Discussion by luckyparadise** [filecoin-project/FIPs#593] [General]
This weekly report provides a high-level overview of what's happening with Filecoin governance.

Anyone is welcome to participate in governance. To learn more about any of the below materials, simply follow the links and feel free to weigh in.

If you have questions or ideas related to FIPs or Filecoin governance, please start a new discussion in the FIPs Forum or post directly in [#fil-fips](https://github.com/filecoin-project/FIPs/discussions/filecoinproject.slack.com).

---

**Accepted FIPs** - *FIPs awaiting network implementation*

- FIP0029 - Beneficiary address for storage providers [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0029.md)[(Discussion)](https://github.com/filecoin-project/FIPs/issues/213)
- FIP0047 - Proof Expiration & PoRep Security Policy [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0047.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/415)
- FIP0050 - API between user-programmed actors and built-in actors [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0050.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/401)

**Last Call** - *Final opportunity for community feedback!*

- FIP0048 - f4 Address Class [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0048.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/473)
- FIP0052 - Increase max sector commitment to 3.5 years [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0052.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/475)

**Current FIP Drafts** - *Now accepting community feedback; merged into repo*

- FIP0037 - Gas Model Adjustment for User Programmability [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0037.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/316)
- FIP0039 - Filecoin Message Replay Protection [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0039.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/301)
- FIP0049 - Actor Events [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0049.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/484)
- FIP0055 - Supporting Ethereum Accounts, Addresses, and Transactions [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0055.md)[(Discussion 1)](https://github.com/filecoin-project/FIPs/discussions/388)[(Discussion 2)](https://github.com/filecoin-project/FIPs/discussions/490)[(Discussion 1)](https://github.com/filecoin-project/FIPs/discussions/287)

**Open Pull Requests** - *upcoming FIP drafts awaiting technical review; now accepting community feedback*

- Update gas charging schedule and system limits for FEVM[(PR)](https://github.com/filecoin-project/FIPs/pull/573)
- Sector Duration Multiplier (SDM) community consideration - [(PR)](https://github.com/filecoin-project/FIPs/pull/571)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/554)
- FIP0054: Filecoin EVM runtime (FEVM) - [(PR)](https://github.com/filecoin-project/FIPs/pull/569)
- Initial draft for Verifiable Data Aggregation - [(PR)](https://github.com/filecoin-project/FIPs/pull/568)
- FRC: non-normative FEVM architectural overview - [(PR)](https://github.com/filecoin-project/FIPs/pull/520)
- FIP0043: Introduce Domain Separation - [(FIP)](https://github.com/filecoin-project/FIPs/pull/423/files)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/387)
- Change ProofRefreshWindow value to 60 days - [(PR)](https://github.com/filecoin-project/FIPs/pull/590) 

 **:fire: HOT :fire: Discussion Topics**

- @Kaitlin Beegle has shared proposed updates to FIP001. One of the objectives is to formalize the establishment of a Governance Working Group. Please share your thoughts and read the post in detail [here](https://github.com/filecoin-project/FIPs/discussions/585)
- @vkalghatgi started a discussion thread aimed at revisiting a Sector Duration Multiplier. Read more about the scenarios being proposed for the SDM Policy. You can read in detail and drop your comments/questions [here](https://github.com/filecoin-project/FIPs/discussions/554)
- @adam-dltx started a discussion topic on the challenge of liquidity required to scale as a Storage Provider (SP). Please read in detail and share your thoughts and comments [here](https://github.com/filecoin-project/FIPs/discussions/583)
- @anorth posted a discussion on a proposal aimed at sperating proof fees from execution gas by removing the batch balancer mechanism. Please read more and contribute [here](https://github.com/filecoin-project/FIPs/discussions/587)

**Other Governance Updates**
* Plans are rapidly unfolding for nv18 - to stay in the loop, be sure to check out the [planning thread in the TPM repo](https://github.com/filecoin-project/tpm/discussions/115)!

**Comment by steven004** [comment]
@luckyparadise ，you can remove FIP-0029 from the list, since it is already in NV18, so not awaiting implementation.
